### PR TITLE
update footer

### PR DESCRIPTION
**Pull request by emillipede**
Hi @amercader 

I tried to make a couple of small changes to the website footer. I'm not very familiar with HTML, so I'm not certain I added the link in the correct way.

Please let me know.

Thank you,
Emily

### CODE REVIEW

File: ckanext/lacounts/templates/footer.html [hunk 1]
```diff
@@ -13,9 +13,9 @@
       <a class="logo" href="{{ h.url_for('home.index') }}"><img src="{% url_for_static '/base/images/la-counts-logo.png' %}" alt="{{ g.site_title }}" title="{{ g.site_title }}" /></a>
       <div class="footer-content">
         <address>
-          221 S. Figueroa Street #400<br/>
+          1000. N. Alameda Street, Suite 420 <br/>
           Los Angeles, CA 90012<br/>
-          Email: <a href="mailto:hello@lacounts.org">hello@lacounts.org</a><br/>
+          LA Counts is a project of <a target="_blank" rel="noopener noreferrer" href = "http://www.communitypartners.org">Community Partners</a><br/>
```
Comment (amercader) [inline ckanext/lacounts/templates/footer.html:18]:
```suggestion
          LA Counts is a project of <a target="_blank" rel="noopener noreferrer" href="http://www.communitypartners.org">Community Partners</a><br/>
```

File: ckanext/lacounts/templates/footer.html [hunk 2]
```diff
@@ -30,10 +30,7 @@
           <a href="{{ h.url_for(controller='ckanext.lacounts.controller:StaticController', action='privacypolicy') }}">{{ _('Privacy Policy') }}</a>
         </nav>
         <div>
-          <ul class="platforms">
-            <li>
-              <a href="https://www.linkedin.com/company/lacounts/" target="_blank"><i class="fa fa-linkedin-square" aria-hidden="true"></i><span class="sr-only">LinkedIn</span></a>
-            </li>
+          <ul class="platforms"
```
Comment (amercader) [inline ckanext/lacounts/templates/footer.html:33]:
```suggestion
          <ul class="platforms">
```

**Review by amercader — COMMENTED** [review]
These look great @emillipede , I only found a couple of small typos. You can apply the suggested fixes clicking on the "Commit suggestion" button. After these are applied I'll merge them and deploy.

**Comment by emillipede** [comment]
Thank you @amercader !

**Comment by amercader** [comment]
@emillipede this is now deployed to the live site

**Comment by emillipede** [comment]
thank you again @amercader !

**Pull request merged**